cmd/ackdev/cmd/setup.go

```diff
@@ -0,0 +1,69 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws-controllers-k8s/dev-tools/pkg/config"
+)
+
+var (
+	optSetupRootDirectory   string
+	optSetupInitialServices string
+)
+
+func init() {
+	setupCmd.PersistentFlags().StringVar(&optSetupRootDirectory, "root-directory", defaultRootDirectory, "root directory for ACK repositories")
+	setupCmd.PersistentFlags().StringVarP(&optSetupInitialServices, "services", "s", "", "services injected in the generated configuration file")
```

Comment:
Why use `StringVar` for one and `StringVarP` for the other?

Reply:
No strong reasons behind that, i just couldn't find one letter that represents `root-directory`. Previously it was `-R` .. WDYT @RedbackThomson ?

Reply:
Oh yeah just keep it long-form, then, I suppose. Nbd